chopper/test/ensure_build_test.dart

```diff
@@ -0,0 +1,16 @@
+@TestOn('vm')
+@Timeout(Duration(seconds: 120))
+import 'package:build_verify/build_verify.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test(
+    'ensure_build',
+    () => expectBuildClean(
+      packageRelativeDirectory: 'chopper',
+      gitDiffPathArguments: [
+        'test/test_service.chopper.dart',
+      ],
+    ),
+  );
+}
```

Comment:
Is it intended to test it "twice". Since its also added in the `chopper_generator`? I would think it makes sense if the verify only takes place at one place. In this case the the `chopper_generator` since we want to be sure no unexpected changes happen.

Reply:
Well, the thing is that both packages can have their own `test/test_service.dart` files which could be different. This one here in `chopper_generator` is just to test the generation of various methods.

I tried running a test from within `chopper_generator` referencing `../chopper/test/test_service.chopper.dart` directly, but the test wouldn't run correctly. And rather than using dodgy symlinks, I opted for a copy.  At the moment I think it's "good enough" 🙈 to copy the one from `chopper/tests/test_service.dart` to here. Some tests always beat no tests IMHO 🍡